src/server/api/schema.js

```diff
@@ -901,11 +906,15 @@ const rootMutations = {
 
     createOptOut: async (
       _,
-      { optOut, campaignContactId },
+      { organizationId, optOut, campaignContactId },
```

Comment:
While re-reviewing this after we encountered the optout (inconsistent) failure, I realized that we do `const campaign = await loaders.campaign.load(contact.campaign_id);` below this call.

It seems better to bring that call up higher, and then get organizationId from campaign.organization_id -- That way we avoid an edge-case privilege escalation in a multi-tenant situation, where an ADMIN of another organization could technically optout any contact from any other organization.

It also seems like it could reduce some of the complexity -- we don't need to add the organizationId to the api call, mutation, etc.

This isn't high priority relative to the bug, but if it helps to simplify the PR to reduce the bug surface area, then this seems like an opportunity.